source/fundamentals/crud/read-operations/cursor.txt

```diff
@@ -13,13 +13,13 @@ Access Data From a Cursor
 Overview
 --------
 
-In this guide, you can learn how to access data using a cursor with the
+In this guide, you can learn how to access data using a **cursor** with the
 MongoDB Java driver. 
 
-Read operations that return multiple documents do not immediately return
-all values matching the query. Because a query can potentially match
-a large number of documents, we need to be able to access or store the
-matched documents. 
+A cursor is a tool for iterating over database results while only holding a
```

Comment:
`tool` is a strange word to use in this context.  Can you re-word?  Perhaps:

    A cursor allows an application to iterate ....

Reply:
Thanks for the note. I agree "tool" is a strange word, however I would like to try and give readers a description of what a cursor "is" outside of what a cursor "allows". I know when I was first studying database cursors, I had trouble understanding them because a lot of explanations explained them as "you can use a cursor to perform X" rather than giving me a concrete description of the form "a cursor is a ...".

Are any of the following replacements for "tool" acceptable?

- A cursor is a software construct that allows...
- A cursor is a mechanism that allows...
- A cursor is two objects, one within the driver and one within the database, that allows...
- A cursor is an object and protocol that allows...

Just trying to give readers a noun to associate with a cursor. This has been very helpful for me in learning new ideas in the past, and is something we have tried to incorporate into definitions of terms in other sections of the docs.

Reply:
Mechanism is better.  It's used on https://en.wikipedia.org/wiki/Cursor_(databases) as well FWIW.

Reply:
Updated to mechanism! Feel free to take another look when you are available.